### ISSUE TITLE
Log axis if distribution is log scale

### ISSUE THREAD

**Issue by Koowah** [optuna/optuna-dashboard#576]
### Description

Hi,

I found another similar issue #64 that has apparently been fixed in #348 , but it doesn't appear to work on my end as shown below (x axis is learning rate log scale using `"lr": trial.suggest_float('learning_rate', 1e-4, 1e-1, log=True)`. Any idea why that might be ?

Thanks 😄 
![image](https://github.com/optuna/optuna-dashboard/assets/81920095/1688e49f-2e5d-4b89-a18d-29962b038d8b)
![image](https://github.com/optuna/optuna-dashboard/assets/81920095/4a780340-cf56-46aa-80c1-e2d195e82f1a)

### How to Reproduce

`"lr": trial.suggest_float('learning_rate', 1e-4, 1e-1, log=True)` + execute optuna dashboard

### Python version

3.11.4

### Optuna version

3.3.0

### optuna-dashboard version or git revision

0.12.0

### Web browser

Google Chrome

**Comment by Koowah** [comment]
Let me know if having the study that gave these results would help in any way.

**Comment by beyarkay** [comment]
I'd also really appreciate this feature.

**Comment by keisuke-umezawa** [comment]
It will be implemented in https://github.com/optuna/optuna-dashboard/pull/611

**Comment by Koowah** [comment]
Thank you @keisuke-umezawa 😄 

What about log scale of parallel coordinates plot ? It was added to optuna in [#2164](https://github.com/optuna/optuna/pull/2164) (and works properly using `optuna.visualization`) but doesn't work in optuna dashboard as shown above.

**Comment by Koowah** [comment]
@keisuke-umezawa 
Apologies for posting here eventhough the issue is closed.
Are these two PRs included in the pre-release ?
Thank you
